datasource/src/test/scala/org/locationtech/rasterframes/datasource/geotrellis/GeoTrellisDataSourceSpec.scala

```diff
@@ -48,7 +48,8 @@ import org.scalatest.{BeforeAndAfterAll, Inspectors}
 
 import scala.math.{max, min}
 
-class GeoTrellisDataSourceSpec extends TestEnvironment with BeforeAndAfterAll with Inspectors with DataSourceOptions {
+trait GeoTrellisDataSourceSpec extends TestEnvironment with BeforeAndAfterAll with Inspectors with DataSourceOptions {
```

Comment:
Very little motivation to fix this. Note that the push-down predicates also got busted with move to spark 3.2. 

Can't imagine ever using this rather than reading COGs or even Zarr. Unless somebody would like to fix this I would like to suggest to remove this feature altogether to keep things tidy.

Reply:
@metasim @pomadchin What do you guys think?

Reply:
:+1: